x-pack/filebeat/module/threatintel/_meta/kibana/7/lens/b0837690-df52-11eb-8f2b-753caedf727d.json

```diff
@@ -28,7 +28,7 @@
                                         "size": 10
                                     },
                                     "scale": "ordinal",
-                                    "sourceField": "recordedfuture.risk.criticalityLabel"
+                                    "sourceField": "recordedfuture.criticality_label"
```

Comment:
Is this field defined? I can't find it referenced anywhere else. Should there be a rename/copy from `recordedfuture.evidence_details[:].CriticalityLabel`?